drivers/user_permission_report/app/views/user_permission_report/warehouse_reports/reports/index.haml

```diff
@@ -52,18 +52,21 @@
               = link_to user_permission_report_warehouse_reports_user_path(user), data: {loads_in_pjax_modal: true}, class: 'btn btn-xs btn-primary btn-muted btn-hmis' do
                 View Details
             %td
-              - if user.roles.count == Role.count
-                All Roles
+              - if user.using_acls?
+                TODO
```

Comment:
Do we need this report as part of migration support?

Reply:
Yes, postponing for the moment.